wallets/react-wallet-v2/package.json

```diff
@@ -39,22 +39,24 @@
     "mnemonic-keyring": "1.4.0",
     "near-api-js": "^0.45.0",
     "next": "12.1.5",
+    "permissionless": "0.0.0-main.20231120T173047",
     "react": "17.0.2",
     "react-code-blocks": "0.0.9-0",
     "react-dom": "17.0.2",
     "react-hot-toast": "^2.4.1",
     "react-qr-reader-es6": "2.2.1-2",
     "solana-wallet": "^1.0.2",
     "tronweb": "^4.4.0",
-    "valtio": "1.6.0"
+    "valtio": "1.6.0",
+    "viem": "^1.19.13"
```

Comment:
nit: I'm not sure about this repo exactly, but we tend to avoid using `^` and other "relaxed" modifiers in npm versions and instead prefer to use strict versioning. Might be good idea to apply here (needs to remove package lock file and re-install dependencies fresh)